src/android/jni/jni_user.cpp

```diff
@@ -0,0 +1,121 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+
+#include <jni.h>
+#include "error.h"
+#include "../../../include/librealsense2/rs.h"
+#include "../../../include/librealsense2/hpp/rs_frame.hpp"
+
+#include "jni_logging.h"
+#include "jni_user.h"
+
+bool rs_jni_callback_init(JNIEnv *env, jobject jcb, rs_jni_userdata* ud)
```

Comment:
Please add here some comment about the aim of this file "jni_user" (here or/and in the header file).
If it is only for the callback, this should be named accordingly.
If this is for other purposes also, a comment would help us to understand the design and go on with it together.